tests/framework/results.go

```diff
@@ -96,7 +116,27 @@ func WriteContent(resultsFile *os.File, content string) error {
 	return nil
 }
 
-// NewCSVEncoder returns a vegeta CSV encoder.
-func NewCSVEncoder(w io.Writer) vegeta.Encoder {
+// NewVegetaCSVEncoder returns a vegeta CSV encoder.
+func NewVegetaCSVEncoder(w io.Writer) vegeta.Encoder {
```

Comment:
I think the reason we kept this agnostic is if we ever change the test framework, we don't have to update the function calls.